packages/npm-mongo/package.js

```diff
@@ -3,12 +3,12 @@
 
 Package.describe({
   summary: "Wrapper around the mongo npm package",
-  version: "3.1.2",
+  version: "3.3.0",
   documentation: null
 });
 
 Npm.depends({
-  mongodb: "3.1.13"
+  mongodb: "3.3.2"
```

Comment:
Can you update the shrinkwrap file for the `npm-mongo` package by running `meteor test-packages ./npm-mongo` in the `meteor/packages` directory?